sdk/compute/Azure.ResourceManager.Compute/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Release History
 
-## 1.2.0-beta.3 (Unreleased)
+## 1.2.0 (2023-09-15)
```

Comment:
Please update the version in Azure.ResourceManager.Compute.csproj as well

Reply:
done. thank you